src/common/gui/editor/Editor.ts

```diff
@@ -128,6 +129,10 @@ export class Editor implements ImageHandler, Component {
 		this.showOutline = show
 	}
 
+	setAreRelativeLinksAllowed(areRelativeLinksAllowed: boolean) {
```

Comment:
I thought it might be better to just pass it in constructor and not have a setter but we do have `insertHTML()` anyway so there's not really much of a security benefit. So I guess it's fine as is.